RentItAll/src/items/ItemDBC.java

```diff
@@ -0,0 +1,149 @@
+package items;
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.util.List;
+import java.util.ArrayList;
+import java.sql.Statement;
+import java.sql.ResultSet;
+
+//data base management class for items table
+public class ItemDBC {
+	private static String jdbcURL;
+	private static String jdbcUsername;
+    private static String jdbcPassword;
+    private static Connection jdbcConnection;
+     
+    public ItemDBC(String jdbcURL, String jdbcUsername, String jdbcPassword) {
```

Comment:
Ome way to do is to create a default contsructor and create a separate properties files to define the jdbc url, username or password. 
Otherwise you can directly assign values

public ItemDBC(){
this.jdbcURL = " Manually define here the url"
}